installer/PowerToysSetup/PowerToys.wxs

```diff
@@ -113,6 +123,22 @@
                     Size="$(var.WinAppSDKPayloadSize)"
                     Version="1.0.3.0"
                     Hash="$(var.WinAppSDKPayloadHash)" />
+            </ExePackage>            
+            <ExePackage
+                Name="VCRedist.exe"
```

Comment:
Can we use other name here, like `VCRedist-14.32.31326.exe`? The main reason for this is that if there's a VCRedist.exe in the same path as the installer, this will try to use that one and fail.
Also is there some code we can use to detect if we get the "more recent version is installed" error and still continue?

Reply:
Done. Yes, we already do that for a minor version, see `DetectCondition="DetectedVCRedistVersion >= 32"`. And we shouldn't be checking for a greater major version, since it'll probably ABI-incompatible.